CMake/External_Qt.cmake

```diff
@@ -132,7 +132,7 @@ if(WIN32)
     endif()
   endif()
 else()
-  set(Qt_configure ./configure)
+  set(Qt_configure PKG_CONFIG_PATH=${fletch_BUILD_INSTALL_PREFIX}/lib/pkgconfig ./configure)
```

Comment:
I wouldn't disbelieve this works for you, and it *might* always work, but we might want to do more like we do for the build command, i.e. hoist `set(env ${CMAKE_COMMAND} -E env)` to earlier and have a `build_env_var` and `configure_env_var`? Also, we should probably preserve an existing `PKG_CONFIG_PATH`...